Examples/Scripts/Python/simulate_barrel.py

```
@@ -0,0 +1,90 @@
+import acts
+import argparse
```

Ruff (0.11.9)

`argparse` imported but unused

Remove unused import: `argparse`

(F401)